sample-apps/django_sample/blog/tests.py

```diff
@@ -17,3 +19,13 @@ def test_app_logging(self):
         self.assertEqual(msg_log, "Hello, world")
         self.assertEqual(response.status_code, 200)
         self.assertEqual(b"Judoscale Sample App", response.content)
+
+    def test_reporter_starts_for_each_process(self):
+        with self.assertLogs() as captured:
+            response = self.client.get('/')
+
+            self.assertEqual(response.status_code, 200)
+            # TODO patch the request header to return HTTP_X_REQUEST_START for test client
+            self.assertEqual(reporter.is_running, True)
+            print("METRICS", reporter.get_metrics())
+            self.assertEqual(len(reporter.get_metrics()), 0)
```

Comment:
I noticed that when running the tests, because there is no proxy server, but just the test client, we don't get metrics, I need to patch this for the test to give a length other than 0. But I can do it in a later PR. Let me know.

Reply:
I'd prefer a later PR. 👍